launch/camera2.launch

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+<launch>
+
+    <!-- Produced topics -->
+    <arg name="object_markers"  default="object_markers"/>
+
+    <!-- Options -->
+    <arg name="use_cuda"            default="true"/>
+    <arg name="enable_bbox_image"   default="false"/>
+    <arg name="enable_aruco"        default="false"/>
+    <arg name="is_sim"              default="true"/>
+    <arg name="map"         default="floor2"/>
+
+    <!-- Auxiliary -->
+    <arg name="max_age"             default="30"/>
+    <arg name="aruco_size"          default="0.360"/>
+    <rosparam command="load" file="$(find svea_core)/params/obstacles.yaml" />
+
+    <!-- Constants -->
+    <arg name="depth_image"     value="/zed/zed_node/depth/depth_registered"/>
+    <arg name="image"           value="/zed/zed_node/rgb/image_rect_color"/>
+    <arg name="websocket_port"  value="9090"/>
+
+
+    <arg name="map_img_name" default="$(find svea_core)/maps/floor2.yaml"/>
+    <node pkg="map_server" name="map_server" type="map_server" args="$(arg map_img_name)"/>
+
+    <!-- Nodes -->
+    <include file ="$(find el2425_standards)/launch/infra1.launch"/>
```

Comment:
remove (course specific launch)

Reply:
Nice, I have left all those files to make sure we don't need them